src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -2230,8 +2268,27 @@ in src-script.js and main.js
 		margin: 0;
 		padding: var(--nav-sub-mobile-padding);
 	}
+
+	.example-wrap.scraped-example {
+		flex-wrap: wrap;
+	}
+	.example-wrap .scraped-example-title {
+		width: 100%;
+	}
+	.example-wrap.scraped-example .rust {
+		/* Dirty hacky to force it to remain on the same line as the line numbers. */
+		width: 10px;
+	}
+	.example-wrap.scraped-example .button-holder {
+		top: calc(var(--pre-line-height) + 4px);
+	}
+	.scraped-example:not(.expanded)::before {
+		/* The gradient effect needs to be moved under the title */
```

Comment:
I didn't like this change and yesterday I thought about another way of doing it which would work with line-wrap as well. Gonna do it instead of this solution.